### PR TITLE
add this pointer to wasapi logs to help identify source in logs

### PR DESCRIPTION
**Pull request by summeroff**

### CODE REVIEW

File: plugins/win-wasapi/win-wasapi.cpp [hunk 1]
```diff
@@ -252,13 +252,13 @@ HRESULT WASAPISource::InitDevice(IMMDeviceEnumerator *enumerator)
 		return res;
 
 	if (!device_name.empty()) {
-		blog(LOG_INFO, "[WASAPISource::InitDevice]: Failed to init device and device name not empty",
-		     device_name.c_str());
+		blog(LOG_INFO, "[WASAPISource::InitDevice][%08X]: Failed to init device and device name not empty",
+		     this, device_name.c_str());
```
Comment (EddyGharbi) [inline plugins/win-wasapi/win-wasapi.cpp:256]: Should we remove `device_name.c_str()`? It's not being logged no?

Comment (summeroff) [reply plugins/win-wasapi/win-wasapi.cpp:256]: fixed. made it to print device name.

File: plugins/win-wasapi/win-wasapi.cpp [hunk 2]
```diff
@@ -252,13 +252,13 @@ HRESULT WASAPISource::InitDevice(IMMDeviceEnumerator *enumerator)
...
-		     device_name.c_str());
+		blog(LOG_INFO, "[WASAPISource::InitDevice][%08X]: Failed to init device and device name not empty %s",
+		     this, device_name.c_str());
 		devices.clear();
 		GetWASAPIAudioDevices(devices, isInputDevice, device_name);
 		if (devices.size()) {
-			blog(LOG_INFO, "[WASAPISource::InitDevice]: Use divice from GetWASAPIAudioDevices",
-			     device_name.c_str());
+			blog(LOG_INFO, "[WASAPISource::InitDevice][%08X]: Use divice from GetWASAPIAudioDevices",
+			     this, device_name.c_str());
```
Comment (EddyGharbi) [inline plugins/win-wasapi/win-wasapi.cpp:261]: Same issue as above, the device name is not being printed

Comment (summeroff) [reply plugins/win-wasapi/win-wasapi.cpp:261]: done

**Review by EddyGharbi — APPROVED** [review]

**Pull request merged**